mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.ui/src/org/eclipse/mylyn/gitlab/ui/GitlabImages.java

```diff
@@ -0,0 +1,85 @@
+package org.eclipse.mylyn.gitlab.ui;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.jface.resource.ImageRegistry;
+import org.eclipse.mylyn.commons.ui.CommonImages;
+import org.eclipse.mylyn.internal.commons.ui.CommonsUiConstants;
+import org.eclipse.swt.graphics.Image;
+import org.osgi.framework.Bundle;
+
+public class GitlabImages {
+
+    private static final URL baseUrl;
+    static {
+	Bundle bundle = Platform.getBundle(GitlabUiActivator.PLUGIN_ID);
+	if (bundle != null) {
+	    baseUrl = bundle.getEntry("/icons/"); //$NON-NLS-1$
+	} else {
+	    URL iconsUrl = null;
+	    try {
+		// lookup location of CommonImages class on disk
+		iconsUrl = new URL(GitlabImages.class.getResource("GitlabImages.class"), "../../../../../../icons/"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
do we really consider operations without OSGi running?

Reply:
removed with next commit.